models/articles.model.js

```diff
@@ -41,20 +41,3 @@ exports.selectAllArticles = () => {
       next(error);
     });
 };
-
-exports.selectArticleCommentsByArticleId = (article_id) => {
-  let queryString = `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`;
-
-  return db.query(queryString, [article_id]).then(({ rows }) => {
-    return rows;
-  });
-};
-exports.checkArticleExists = (article_id) => {
```

Comment:
Not sure why in this PR this is shown as deleted. But I'll leave the comments here for it anyway

This is a good idea! However, article_id and author are foreign keys, so PSQL will throw an error (that you can handle with a [PSQL error handler](https://notes.northcoders.com/courses/js-back-end/error-handling#psql-thrown-errors)) if you try and insert a non-existent or invalid value.
This means you don't need to check separately (comment these checks out and run the code to see what I mean - if you have PSQL error handlers, they will catch this, otherwise you can console.log the error code)